src/python/dxpy/cli/org.py

```diff
@@ -242,14 +242,17 @@ def _get_org_update_args(args):
     if args.saml_idp is not None:
         org_update_inputs["samlIdP"] = args.saml_idp
 
```

Comment:
```suggestion
if args.enable_job_reuse and args.disable_job_reuse:
    raise DXCLIError("--enable-job-reuse and --disable-job-reuse cannot be used simultaneously")

```

Reply:
This should be handled by argparse since they are mutually exclusive
`update_job_reuse_args = parser_update_org.add_mutually_exclusive_group(required=False)`

Reply:
Ah, cool!